libs/blocks/quiz/quiz.css

```diff
@@ -665,64 +770,92 @@ TODO: Need to check if we can skip this somehow.
   opacity: 0.5;
 }
 
-.card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option {
-  background-color: hsla(0, 0%, 98%, .8);
-  min-width: 9.625rem;
-  min-height: 13.75rem;
-  width: 9.625rem;
-  margin: 0.9375rem;
-  box-shadow: 10px 10px 6px 0 rgb(0 0 0 / 16%);
-  border: 1px solid #fff;
-  border-radius: 0.5rem;
-  cursor: pointer;
-  height: 10rem;
+@media screen and (min-width: 900px) {
+  .card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option {
+    background-color: hsla(0, 0%, 98%, .8);
```

Comment:
I see a lot of ways of defining colours - hex, names, rgb, hsla, could they be consolidated to follow a pattern?